packages/legend-extension-dsl-data-space/src/graph-manager/protocol/pure/v1/transformation/pureProtocol/V1_DSL_DataSpace_ProtocolHelper.ts

```diff
@@ -43,19 +44,24 @@ import {
 } from '@finos/legend-shared';
 import {
   type V1_DataSpaceSupportInfo,
+  type V1_DataSpaceExecutable,
   V1_DataSpace,
   V1_DataSpaceExecutionContext,
   V1_DataSpaceSupportEmail,
   V1_DataSpaceSupportCombinedInfo,
-  V1_DataSpaceExecutable,
   V1_DataSpaceDiagram,
   V1_DataSpaceElementPointer,
+  V1_DataSpaceTemplateExecutable,
+  V1_DataSpacePackageableElementExecutable,
 } from '../../model/packageableElements/dataSpace/V1_DSL_DataSpace_DataSpace.js';
 import { V1_MappingIncludeDataSpace } from '../../model/packageableElements/mapping/V1_DSL_DataSpace_MappingIncludeDataSpace.js';
 
 export const V1_DATA_SPACE_ELEMENT_PROTOCOL_TYPE = 'dataSpace';
 const V1_DATA_SPACE_SUPPORT_EMAIL_TYPE = 'email';
 const V1_DATA_SPACE_SUPPORT_COMBINED_INFO_TYPE = 'combined';
+const V1_DATA_SPACE_PACKAGEABLEMENT_EXECUTABLE =
```

Comment:
Nit: looks like this is a typo
```suggestion
const V1_DATA_SPACE_PACKAGEABLE_ELEMENT_EXECUTABLE =
```

Reply:
thanks!